frontend/src/components/Common/SwipeCard.css

```diff
@@ -20,6 +20,38 @@
   border-radius: 30px;
 }
 
+.CardOverlayText {
+    top: 2em;
+    left: -5em;
+    width: em;
+    color: #fff;
+    position: absolute;
+    text-align: center;
+    visibility: hidden;
+    letter-spacing: .06em;
+    padding: 0.5em 5em 0.4em 5em;
+    text-shadow: 0 0 0.75em #444;
+    box-shadow: 0 0 0.5em rgba(0,0,0,0.5);
+    font: bold 1em Arial, Sans-Serif;
```

Comment:
Hi David, just a small change,
Going through the new styling of the frontend, we started using: 
`font-family:monospace;`
I think it would be better to use this font instead of Arial for consistency.

Reply:
Oh, its just what we've been using for now I don't think we compared many fonts.

Either is completely fine but for now I think its best to be consistent, and we can change them all the fonts together at a later date if we feel we need to.